packages/dapp/src/pages/EscrowInfo.jsx

```diff
@@ -81,19 +83,25 @@ export const EscrowInfo = () => {
   };
 
   const getSmartInvoiceData = async () => {
-    if (
-      utils.isAddress(context.invoice_id) &&
-      !Number.isNaN(parseInt(context.chainID))
-    ) {
-      let smartInvoice = await getSmartInvoiceAddress(
-        context.invoice_id,
-        context.provider
-      );
-      console.log(context.chainID, smartInvoice);
-      getInvoice(parseInt(context.chainID), smartInvoice).then((i) => {
-        console.log(i);
-        setInvoice(i);
-      });
+    try {
+      if (
+        utils.isAddress(context.invoice_id) &&
+        !Number.isNaN(parseInt(context.chainID))
+      ) {
+        let smartInvoice = await getSmartInvoiceAddress(
+          context.invoice_id,
+          context.provider,
+        );
+        console.log(context.chainID, smartInvoice);
```

Comment:
Do we need these console logs ?

Reply:
Yup. that was actually added to see if the invoice is fetched if incase something isn't working for the cleric training session.